cedar-lean/DiffTest/Parser.lean

```diff
@@ -246,6 +261,26 @@ partial def exprToValue : Expr → ParseResult Value
 def jsonToValue (json : Lean.Json) : ParseResult Value :=
   jsonToExpr json >>= exprToValue
 
+
+inductive ValueOrExpr where
+  | value : Value → ValueOrExpr
+  | expr : Cedar.Partial.Expr  → ValueOrExpr
+
+def jsonToValueOrExpr (json : Lean.Json) : ParseResult ValueOrExpr := do
+  match json.getObjVal? "Value" with
+    | .ok v => ValueOrExpr.value <$> (jsonToValue v)
+    | .error _ => match json.getObjVal? "Expr" with
+      | .ok e => ValueOrExpr.expr <$> (jsonToPartialExpr e)
+      | .error _ => .error "Expected either `expr` or `value`"
```

Comment:
```suggestion
      | .error _ => .error "jsonToValueOrExpr: expected either `Expr` or `Value`"
```

Reply:
did you reject this suggestion? it was resolved but it doesn't appear in the latest diff